### PR TITLE
Refactor gopro

### PR DESCRIPTION
**Pull request by daktak**
Generic function to copy .

Please review and advise.

### CODE REVIEW

**Comment by KonradIT** [comment]
Hm switch the PR so it points to `golanci-linters`, I already refactored the gopro.go code a bit, and will merge PR #39 with your changes in it.

File: pkg/gopro/gopro.go [hunk 1]
```diff
@@ -1046,3 +732,53 @@ func getFileTime(osPathname string, utcFix bool) time.Time {
...
+
+func getMediaDate(d time.Time, sortoptions SortOptions) string {
+	mediaDate := d.Format("02-01-2006")
+	if strings.Contains(sortoptions.DateFormat, "yyyy") && strings.Contains(sortoptions.DateFormat, "mm") && strings.Contains(sortoptions.DateFormat, "dd") {
+		mediaDate = d.Format(replacer.Replace(sortoptions.DateFormat))
+	}
+        return mediaDate
+}
+
+func parse(folder string, name string, de *godirwalk.Dirent, osPathname string, sortoptions SortOptions, result utils.Result) utils.Result {
```
Comment (KonradIT) [inline pkg/gopro/gopro.go]: `de *godirwalk.Dirent` is unused in the function.

**Review by KonradIT — CHANGES_REQUESTED** [review]

File: pkg/gopro/gopro.go (hunk 1 above)
Comment (daktak) [reply pkg/gopro/gopro.go]: Thanks, removed

**Comment by daktak** [comment]
Updated

**Review by KonradIT — APPROVED** [review]

**Comment by KonradIT** [comment]
Thanks!

**Pull request merged**